src/lib/storage/table.hpp

```diff
@@ -10,6 +10,8 @@
 #include "chunk.hpp"
 #include "proxy_chunk.hpp"
 #include "storage/index/index_info.hpp"
+#include "storage/partitioning/abstract_partition_schema.hpp"
+#include "storage/partitioning/hash_function.hpp"
```

Comment:
Forward declarations instead of includes?

Reply:
That leads to an `error: member access into incomplete type`, if the partition schema is accessed.